scripts/index.js

```diff
@@ -1,8 +1,12 @@
 'use strict';
 
 import React from 'react';
-import router from './router';
+import router, {transitionTo} from './router';
 
 router.run((Handler, state) => {
-  React.render(<Handler {...state} />, document.body);
+  // I know, passing `transitionTo` like this is ugly,
+  // haven't found a better solution yet though.
+  // Since `Explore` component is not in the Router `context`
+  // we have to pass it along, somehow.
+  React.render(<Handler {...state} transitionTo={transitionTo} />, document.body);
```

Comment:
Fixed https://github.com/emmenko/flux-react-router-example/commit/ca93d3b9a07df55e08ffc93f02ef437b5ad039b0

I was missing `contextTypes` apparently, thanks for the tip! :)